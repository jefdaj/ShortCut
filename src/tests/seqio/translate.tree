.
|-- cache
|   |-- each
|   |   `-- 5a0cbc306e
|   |       |-- 68f4ddeb65.faa -> $TMPDIR/exprs/translate/68f4ddeb65.faa
|   |       `-- 68f4ddeb65.faa.args
|   `-- list
|       |-- 0aac195355.txt
|       |-- 1f98704802.txt
|       `-- db35af74d8.txt
|-- exprs
|   |-- list
|   |   |-- 49ae883ffc.faa.list -> ../../cache/list/db35af74d8.txt
|   |   |-- 83c6cf693b.fna.list -> ../../cache/list/0aac195355.txt
|   |   `-- fae5b667b7.str.list -> ../../cache/list/1f98704802.txt
|   |-- load_fna
|   |   `-- f110b4b1c9.fna -> $WORKDIR/data/6803genes.fna
|   |-- str
|   |   `-- a2fd369878.str -> ../../cache/list/1f98704802.txt
|   |-- translate
|   |   |-- 68f4ddeb65.faa
|   |   `-- fddf135b43.faa
|   |-- translate_each
|   |   `-- 283ecf7c50.faa.list
|   `-- union
|       `-- 6c1170e158_620ea9c96e.faa.list
`-- vars
    |-- mapped.faa.list -> ../exprs/translate_each/283ecf7c50.faa.list
    |-- result -> ../exprs/union/6c1170e158_620ea9c96e.faa.list
    `-- single.faa -> ../exprs/translate/fddf135b43.faa

12 directories, 17 files
