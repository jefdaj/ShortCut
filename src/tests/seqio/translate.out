[>BAA16579.2 sds solanesyl diphosphate synthase
MISTTSLFAPVDQDLRLLTDNLKRLVGARHPILGAAAEHLFEAGGKRVRPAIVLLVSRAT
LLDQELTARHRRLAEITEMIHTASLVHDDVVDEADLRRNVPTVNSLFDNRVAVLAGDFLF
AQSSWYLANLDNLEVVKLLSEVIRDFAEGEILQSINRFDTDTDLETYLEKSYFKTASLIA
NSAKAAGVLSDAPRDVCDHLYEYGKHLGLAFQIVDDILDFTSPTEVLGKPAGSDLISGNI
...
]
